base/io.jl

```diff
@@ -131,6 +131,8 @@ data has already been buffered. The result is a `Vector{UInt8}`.
 """
 function readavailable end
 
+function isexecutable end
```

Comment:
should this have a method `isexecutable(::IO) = false`?

Reply:
I think we can skip that for now.